doc/reference/storage_btrfs.md

```diff
@@ -1,6 +1,18 @@
 (storage-btrfs)=
 # Btrfs - `btrfs`
 
+{abbr}`Btrfs (B-tree file system)` is a single-disk file system based on the {abbr}`COW (copy-on-write)` principle.
+COW means that data is stored to a different block after it has been modified instead of overwriting the existing data, reducing the risk of data corruption.
+Unlike other file systems, Btrfs is extent-based, which means that it stores data in contiguous areas of memory.
+
+In addition to basic file system features, Btrfs offers RAID and volume management, pooling, snapshots, checksums, compression and other features.
```

Comment:
Ah, you actually have RAID mentioned here which definitely contradicts the single-disk part.